src/Server/RestRouterHandlers/DatabaseRestRouterHandler.cpp

```diff
@@ -30,7 +30,8 @@ std::pair<String, Int32> DatabaseRestRouterHandler::executeGet(const Poco::JSON:
 {
     String query = "SHOW DATABASES;";
 
-    return processQuery(query);
+    Poco::JSON::Object resp;
+    return {processQuery(query, ([this, &resp](Block && block) { this->buildDatabaseArray(block, resp); }), resp), HTTPResponse::HTTP_OK};
```

Comment:
This line is a bit complex, can we separate `processQuery` call to a separate line ?

Reply:
fixed